DataConnectors/ThycoticCEF.json

```diff
@@ -0,0 +1,115 @@
+{
+    "id": "ThycoticCEF",
+    "title": "Thycotic Secret Server",
+    "publisher": "Thycotic, Inc",
+    "descriptionMarkdown": "Common Event Format (CEF) from Thycotic Secret Server ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Thycotic Secret Server",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Get records create new secret",
+            "query": "CommonSecurityLog | where Activity contains \"SECRET - CREATE\""
+        },
+        {
+            "description" : "Get records where view secret",
+            "query" :"CommonSecurityLog | where Activity contains \"SECRET - VIEW\""
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (Thycotic)",
+            "lastDataReceivedQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nCommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ],
+        "customs": [
+            {
+                "name": "Thycotic Secret Server",
+                "description": "must be configured to export logs via Syslog \n\n   [Learn more about configure Secret Server](https://docs.thycotic.com/ss/10.8.0/events-and-alerts/secure-syslog-cef)"
```

Comment:
We are adding a short link to this page so the product version does not have to be used here. Please update this to: `https://updates.thycotic.net/links.ashx?syslog`.

This link should be live by the end of the week.

Reply:
Sorry correction on link (forgot we are moving to a new system) 😬 

`https://thy.center/ss/link/syslog`